images/runtime/node/generateDockerfiles.sh

```diff
@@ -6,11 +6,13 @@
 
 set -e
 
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && cd .. && pwd )
```

Comment:
nit: `cd ..\..\..`?

Reply:
Sure, will do...but will do in next PR which is dealing with versions.